src/frontend/src/pages/AdminToolsPage/AdminToolsUserManagement.tsx

```diff
@@ -0,0 +1,125 @@
+    } else {
+      setUser(null);
+    }
+  };
+
+  const handleRoleChange = (event: SelectChangeEvent) => {
+    setRole(event.target.value);
+    if (user && event.target.value === user.role) {
+      setIsDisabled(true);
+    } else {
+      setIsDisabled(false);
+    }
+  };
+
+  const handleClick = async () => {
+    setHideSuccessLabel(true);
+    await update.mutateAsync({ userId: user?.userId, role }).catch((error) => {
+      alert(error);
+      throw new Error(error);
+    });
+    setHideSuccessLabel(false);
+  };
+
+  return (
+    <PageBlock title={'User Management'}>
+      <Grid container spacing={2}>
+        <Grid item xs={12} md={8}>
+          <div style={{ backgroundColor: 'ActiveBorder', borderRadius: '25px', height: '40px', display: 'flex' }}>
+            <IconButton type="button" sx={{ p: '10px' }} aria-label="search">
+              <SearchIcon />
+            </IconButton>
+            <Autocomplete
+              disablePortal
+              id="autocomplete"
+              onChange={handleSearchChange}
+              options={data.map((user) => `${fullNamePipe(user)}: ${user.email}`)}
+              sx={{
+                width: '100%',
+                '.MuiOutlinedInput-notchedOutline': { border: 0, borderRadius: '25px', borderColor: 'black' }
+              }}
+              size="small"
+              renderInput={(params) => <TextField {...params} label="Select a User" />}
+            />
+          </div>
+        </Grid>
+        <Grid item xs={12} md={4}>
+          <div
+            style={{
+              backgroundColor: '#ff0000',
+              borderColor: '#0062cc',
+              borderRadius: '25px',
+              height: '40px'
+            }}
+          >
+            <FormControl size="small" sx={{ width: '100%' }}>
+              <InputLabel id="select-label">{!user ? 'Current Role' : user.role}</InputLabel>
+              <Select
+                label={!user ? 'Current Role' : user.role}
+                labelId="select-label"
+                id="role-select"
+                value={role}
+                onChange={handleRoleChange}
+                disabled={!user}
+                sx={{
+                  '.MuiOutlinedInput-notchedOutline': { border: 0, borderRadius: '25px', borderColor: 'black' }
+                }}
+              >
+                <MenuItem value={'ADMIN'}>Admin</MenuItem>
+                <MenuItem value={'LEADERSHIP'}>Leadership</MenuItem>
+                <MenuItem value={'MEMBER'}>Member</MenuItem>
+                <MenuItem value={'GUEST'}>Guest</MenuItem>
+              </Select>
+            </FormControl>
+          </div>
+        </Grid>
+      </Grid>
+      <NERButton sx={{ mt: '2px', float: 'right' }} variant="contained" disabled={isDisabled || !user} onClick={handleClick}>
+        Confirm
+      </NERButton>
+      <h4 hidden={hideSuccessLabel} style={{ color: 'primary' }}>
+        Successfully Updated User
+      </h4>
+    </PageBlock>
```

Comment:
I would look up what a controlled component vs an uncontrolled component is for this ticket. It's just good front-end stuff to know and funnily enough I'm p sure you're using one of each here